### ISSUE TITLE
Gregtech port

### ISSUE THREAD

**Issue by Oddface** [John-Smith-Modded/JSTR-1.7.x#54]
I've gone through the older repos and ported pretty much everything with a playable 1.7 version, except this one. The 1.6 folder structure is such a mess and I can't make heads or tails of it. Can someone familiar with the mod take a crack at it?

**Comment by Glowstrontium** [comment]
Ugh, this one is a nightmare - Greg has yet again redone the way the textures work - most of the machines and blocks now use overlay textures, which means that they need completly redoing (and I've found that overlay-based textures don't tend to work very well with the JS texture style because they don't support semi-transparency.

**Comment by damonmensch** [comment]
I managed to get a couple of things working locally on mine for gregtech,
basically just ingots, nuggets, and crushed ores. Atm I'm focusing more on
bluepower, which is mostly just a port of redpower2 but blue toned instead.
I'll try some more after though, a day or two.

**Comment by damonmensch** [comment]
Well never mind that, just noticed someone else beat me to it. Will
prioritize this now

**Comment by damonmensch** [comment]
When doing the port in 7cd720839eb38ad724078bdaabe54c58fb9c99bb I noticed some of the GUIs nerd to be redone. I'll be working on it every couple of days or so.

**Comment by damonmensch** [comment]
Fixed some of the GUIs if you want to check them out. Still need to test them out to make sure no nasty surprises hiding in them.

damonmensch@164ec952a236818b072fe8d6ccd1b3d32161471c

**Comment by damonmensch** [comment]
Ok question for you folks, assuming anyone else aside from myself is ever reading this topic....

Do we want the hazard signs to be like out modern day industrial style, or a more renaissance/medieval-esque feel?

These are the defaults and the modern version would be fairly similar.
![machine_casing_acidhazard](https://cloud.githubusercontent.com/assets/1498614/6028702/8a0493e2-abb9-11e4-85b9-d51a6561c5b9.png) ![machine_casing_biohazard](https://cloud.githubusercontent.com/assets/1498614/6028706/8d0fd024-abb9-11e4-996c-418425e31b40.png) ![machine_casing_explosionhazard](https://cloud.githubusercontent.com/assets/1498614/6028709/8f9f2768-abb9-11e4-9aca-ffc9d07d097f.png) ![machine_casing_firehazard](https://cloud.githubusercontent.com/assets/1498614/6028710/9204d03e-abb9-11e4-8b47-40bd602ad14c.png) ![machine_casing_frosthazard](https://cloud.githubusercontent.com/assets/1498614/6028712/9438687a-abb9-11e4-8b9d-731b97339913.png) ![machine_casing_magichazard](https://cloud.githubusercontent.com/assets/1498614/6028714/9697b076-abb9-11e4-85c2-b109d107dda9.png) ![machine_casing_noisehazard](https://cloud.githubusercontent.com/assets/1498614/6028716/9a81843c-abb9-11e4-85a8-c1b7226a21e8.png) ![machine_casing_radioactivehazard](https://cloud.githubusercontent.com/assets/1498614/6028718/9cfe0988-abb9-11e4-8b85-f85261adea04.png) ![machine_casing_stripes_a](https://cloud.githubusercontent.com/assets/1498614/6028757/d6012a9e-abb9-11e4-92ef-c00ca80188ea.png) ![machine_casing_stripes_b](https://cloud.githubusercontent.com/assets/1498614/6028744/c17f7594-abb9-11e4-83ea-ea29e4dda685.png)

For a medieval/renaissance version I'll make them to how I would feel they would look if created in that era.

**Comment by ScottKillen** [comment]
I like the medieval look, but I know some of the new textures for chisel contributed by @Satellence look modern... do you have a sample maybe?

**Comment by lesdmark** [comment]
So this is going to sound anachronistic but I would make it look like medieval invented modern tech. almost steampunk but not quite.

**Comment by damonmensch** [comment]
That's more how I was gunna lean towards @lesdmark. Borderline medieval and victorian

**Comment by lesdmark** [comment]
Excellent. I think I used those exact two terms to describe something not too long ago.

**Comment by damonmensch** [comment]
GregTech Aspects. I adjusted these to match Thaumcraft's defaults, seeing as JSTR never touched them, and for the most part fit the pack anyway.

Defaults | New | Name (Description)
------------ | ------------- | ------
![electrum](https://cloud.githubusercontent.com/assets/1498614/6113699/3cbe1ede-b067-11e4-9446-e6af60d01b71.png) | ![electrum](https://cloud.githubusercontent.com/assets/1498614/6113713/4cd0d712-b067-11e4-9550-b137307b8d73.png) | Electrum (Electricity)
![magneto](https://cloud.githubusercontent.com/assets/1498614/6113701/3f7b402a-b067-11e4-801b-7002a91db96a.png) | ![magneto](https://cloud.githubusercontent.com/assets/1498614/6113715/4e5fe3a2-b067-11e4-893b-9604e2c2778c.png) | Magneto (Magnetic)
![nebrisum](https://cloud.githubusercontent.com/assets/1498614/6113703/42ad335c-b067-11e4-9e8f-5ddfce280125.png) | ![nebrisum](https://cloud.githubusercontent.com/assets/1498614/6113717/4fb620ae-b067-11e4-806b-0c86305870f3.png) | Nebrisum (Cheatiness)
![radio](https://cloud.githubusercontent.com/assets/1498614/6113705/442abe34-b067-11e4-9815-7faf7fc235e7.png) | ![radio](https://cloud.githubusercontent.com/assets/1498614/6113719/512090be-b067-11e4-9476-6d646a9f34f0.png) | Radio (Radioactive)

**Comment by lesdmark** [comment]
Well done! Those look very good!

**Comment by damonmensch** [comment]
I apologize for disappearing on you guys, work has been hectic with the winter storms in the northeast. I should have some time to work on some textures a bit more in the next couple days

**Comment by goldbattle** [comment]
no problem don't worry about it. I forgot to comment here, and say that it is important that those drop shadows do not get cut off. If these icons ever get used on top of something light in the future it will look horrible. Just something to be careful of. Besides that, thank you for everything so far!

**Comment by damonmensch** [comment]
Yeah I normally avoid such a thing at all costs but noticed almost every single one in the tc4 mod itself has then cut off and since I was making these match those, I didn't worry about it. If there's any plans to do our own version of those icons, let me know

**Comment by goldbattle** [comment]
Ah, ok.

**Comment by draknyte1** [comment]
@damonmensch, I appreciate the work on the GT stuff buddy.

**Comment by mightydanp** [comment]
Can I get an update to the texture? The assembler got 6 input slots instead of 2 and they have added some new machines, single and multiblock.
